src/main/java/org/sert2521/gamename/Log.kt

```diff
@@ -0,0 +1,34 @@
+package org.sert2521.gamename
```

Comment:
In Kotlin, we don't have to define a class if it doesn't need an instance; everything can just be in the file. When it's turned into JVM bytecode, it's your standard utils class with static helper methods. (It'll be called `LogKt` so that's how we use it in Java e.g. `LogKt.logD(...)`. In Kotlin, we just say `logD(...)`)